packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -70,14 +70,29 @@ export async function generateCrudInput(
         const fieldName = prop.name;
         const definedDecorators = morphTsProperty(prop.name, metadata).getDecorators();
         const decorators = [] as Array<string>;
-        if (!prop.nullable) {
-            decorators.push("@IsNotEmpty()");
-        } else {
-            decorators.push("@IsNullable()");
+        let isOptional = prop.nullable;
+
+        if (prop.name != "position") {
```

Comment:
Now that we have the CrudPositionField decorator, do we even need to hardcode position? Shouldn't all fields that use the decorator support positions?